### ISSUE TITLE
[allfonts] ipamona download is broken.

### ISSUE THREAD

**Issue by aearil** [Winetricks/winetricks#1250]
When trying to install `allfonts` the download of ipamona fails and stops the rest of the installer.
The original URL returns a 404:
```
http://www.geocities.jp/ipa_mona/opfc-ModuleHP-1.1.1_withIPAMonaFonts-1.0.8.tar.gz
```
winetricks then tries to download from web archive's with:
```
https://web.archive.org/web/2000/http://www.geocities.jp/ipa_mona/opfc-ModuleHP-1.1.1_withIPAMonaFonts-1.0.8.tar.gz
```
 which redirects to:
```
https://web.archive.org/web/20111108024313/http://www.geocities.jp/ipa_mona/opfc-ModuleHP-1.1.1_withIPAMonaFonts-1.0.8.tar.gz
```
That last URL unfortunately return a 500 error.

However, some later web archive captures work perfectly, for example:
```
https://web.archive.org/web/20190309175311/http://www.geocities.jp/ipa_mona/opfc-ModuleHP-1.1.1_withIPAMonaFonts-1.0.8.tar.gz
```

**Comment by austin987** [comment]
I suspect you have an old version of winetricks. In current winetricks, it's already pulling from archive.org:
https://github.com/Winetricks/winetricks/blob/master/src/winetricks#L12696
` w_download "https://web.archive.org/web/20190309175311/http://www.geocities.jp/ipa_mona/opfc-ModuleHP-1.1.1_withIPAMonaFonts-1.0.8.tar.gz" ab77beea3b051abf606cd8cd3badf6cb24141ef145c60f508fcfef1e3852bb9d`

It was fixed by f98983c9e3304d697f2a456d75cbdfb2eeae1962.

FYI, I'm planning to do a release today/tomorrow, which will include this.

**Comment by aearil** [comment]
Oops, yeah I forgot to provide the version I'm using, sorry.
```
20190310 - sha256sum: cf44b3cbf4134adb9f9dd7eec0d50b45d34ae65c5432d72edc24bcd1841888ba
```
That's the current version from the arch repos.

Thanks for the quick and precise answer :)
